gdscript-completion.el

```diff
@@ -48,6 +49,38 @@
     (list start end gdscript-completion--all-keywords
           . nil)))
 
+(defun gdscript-completion-insert-file-path-at-point (&optional arg)
+  "Insert a file path at point using Godot's relative path (\"res:\").
+
+If Projectile is available, list only the files in the current
+project.  Otherwise, fallback to the built-in function
+`read-file-name'.
+
+If using Projectile, with a prefix ARG invalidates the cache
+first."
+  (interactive "P")
+  (let ((has-projectile (require 'projectile nil 'noerror)))
```

Comment:
I wouldn't call require here, you're loading the projectile package even if the user didn't configure their emacs to use it, as long as they have it somewhere on their load path. You can use the function `featurep` instead:
```lisp
 (let ((has-projectile (featurep 'projectile)))
```